code/client/munkilib/admin/makecatalogslib.py

```diff
@@ -50,7 +49,9 @@ def hash_icons(repo, output_fn=None):
         icon_list.remove('_icon_hashes.plist')
     for icon_ref in icon_list:
         if output_fn:
-            output_fn("Hashing %s..." % (icon_ref))
+            output_fn(
+                colorize.OKGREEN + "Hashing " + colorize.ENDC +
+                "%s..." % (icon_ref))
```

Comment:
If output_fn isn't printing to stdout -- like if it's appending to a log, or sending info to Managed Software Center, what would be the effect of adding these colorization codes?